apps/src/lib/script-editor/ScriptEditor.jsx

```diff
@@ -37,7 +37,7 @@ export default class ScriptEditor extends React.Component {
     loginRequired: PropTypes.bool,
     hideableStages: PropTypes.bool,
     studentDetailProgressView: PropTypes.bool,
-    professionalLearningCourse: PropTypes.bool,
+    professionalLearningCourse: PropTypes.string,
```

Comment:
this is just a bonus prop type fix in response to a console warning I saw when editing the script on levelbuilder.